.editorconfig

```diff
@@ -3,12 +3,15 @@
 
 root = true
 
+[Makefile]
+indent_style = tab
+
 [*]
 end_of_line = LF
+trim_trailing_whitespace = true
 
-[*]
+[*.{hs,lhs}]
 indent_style = space
 indent_size = 4
-trim_trailing_whitespace = true
 insert_final_newline = true
```

Comment:
I wonder if this is a candidate for applying to `*`?

Reply:
```suggestion
```